evals/test_cases/inline_code_completion_tests/variable_tests.py

```diff
@@ -0,0 +1,92 @@
+from evals.eval_types import CodeGenTestCaseCore, InlineCodeCompletionTestCase
+from evals.notebook_states import *
+
+VARIABLE_TESTS = [
+    InlineCodeCompletionTestCase(
+        name="empty_notebook_variable_declaration",
+        test_case_core=CodeGenTestCaseCore(
+            notebook_state=EMPTY_NOTEBOOK,
+            expected_code="x=1",
+            workflow_tags=["variable_declaration"],
+        ),
+        prefix="""
+# Create a variable x and set it equal to 1
+x =""",
```

Comment:
Probably not an issue, but if I were using an inline completer and I had the following line:

```python
# Create a variable x and set it equal to 1
```

I wouldn't even think to type `x = `, I'd just assume the completer would do it.

Reply:
Good point. I made a new eval for that one